Sources/KlaviyoSwift/KlaviyoModels.swift

```diff
@@ -191,9 +208,15 @@ public struct Profile: Equatable {
 }
 
 extension Event.EventName {
-    var value: String {
+    public var value: String {
```

Comment:
@ajaysubra public now!

Reply:
You need to mark the extension as public I think.

Reply:
Thought so too, but swiftlint changed it to this, i do think they are functionally equivalent